src/pre_process/m_global_parameters.fpp

```diff
@@ -392,6 +402,9 @@ contains
             patch_icpp(i)%qv = 0._wp
             patch_icpp(i)%qvp = 0._wp
             patch_icpp(i)%tau_e = 0._wp
+            patch_icpp(i)%Bx = dflt_real
```

Comment:
How aout B(1), B(2), B(3) instead of three scalars? or alternatively B%x, B%y, etc. 

i would like to do this kind of thing across the whole code, and start using that standard for new PRs at least.